bundles/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlActionHandler.java

```diff
@@ -66,8 +67,8 @@ public NikoHomeControlActionHandler(Thing thing) {
     public void handleCommand(ChannelUID channelUID, Command command) {
         NikoHomeControlCommunication nhcComm = getCommunication();
         if (nhcComm == null) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED,
-                    "@text/offline.bridge-unitialized");
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "@text/offline.configuration-error.invalid-bridge-handler");
```

Comment:
This goes back to a remark from @jlaur. You can configure a thing without a bridge (clearly a communication error), and this would result in this code being executed. I would think (and agree with @jlaur) the right thing to do is to put the thing offline and point out the configuration error.

Reply:
Correct. See also #12331.